pygin/pygin/src/far_api.cpp

```diff
@@ -55,6 +55,16 @@ namespace
 		return ULARGE_INTEGER{ ft.dwLowDateTime, ft.dwHighDateTime }.QuadPart;
 	}
 
+	auto UI64ToFileTime(uint64_t ft)
+	{
+		ULARGE_INTEGER i;
+		i.QuadPart = ft;
+		FILETIME result;
+		result.dwLowDateTime = i.LowPart;
+		result.dwHighDateTime = i.HighPart;
```

Comment:
Could be 
```C++
FILETIME UI64ToFileTime(...)
{
    ...
    return { i.LowPart, i.HighPart };
}
```

Reply:
I did not want to rely on the structure layout. I understand that nobody is going to change the layout of Wndows API structures but still, I prefer to see the association 'i.LowPart' <-> 'result.dwLowDateTime' explicitly.
From the reader's standpoint is also helpful to see that there is no unintended fields mismatch here.